netbox_librenms_plugin/templates/netbox_librenms_plugin/_vlan_sync_content.html

```
@@ -16,20 +16,27 @@
 </div>
 {% else %}
 
+{# Migrated donors must not sync: hiding only the button leaves the POST form live (Enter in a filter still submits), so drop the form in migrated mode. #}
+{% if migrated_to_marker %}
+<div>
+{% else %}
 {% with model_name=vlan_sync.object|meta:"model_name" %}
 <form method="post"
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </form> ], start tag match failed [ <form method="post"
      action="{% url 'plugins:netbox_librenms_plugin:sync_selected_vlans' object_type=model_name object_id=vlan_sync.object.pk %}"> ] on line 24.

(tag-pair)